uwblueprint/paramedics-react
# Add custom material ui palette and typography

## Overview
- Added material ui theming for [colours](https://material-ui.com/customization/palette/) and [typography](https://material-ui.com/customization/typography/)
- See [frontend best practices doc](https://www.notion.so/uwblueprintexecs/Front-End-Best-Practices-ba6081af21f14de0bfe3f7d12a757e12#9741409695a145ee9f5dd695a567da22) for how to use the styling changes implemented in this PR
- The palette and typography defined in the theme should match the designers' [component library](https://www.figma.com/file/HAer9VKo9Shchirse9wwgF/Paramedics-Component-Library?node-id=4%3A71)

[Notion ticket](https://www.notion.so/uwblueprintexecs/custom-material-UI-styling-f568bf5aa7ca4a818d7d492b83ebcfb7)

## Changes
- Added material ui theming for colours and typography
- Wrapped the entire app in this theming
- Added `Colours` enum to store colours used in the codebase

## Testing
- Use Material-UI's `Typography` component somewhere and pass the `variant` prop. See front end docs for an example. Make sure it matches the [component library](https://www.figma.com/file/HAer9VKo9Shchirse9wwgF/Paramedics-Component-Library?node-id=4%3A71).
- Try changing the `color` prop on the `Button` component in `AddEventButton.tsx` from "primary" to "secondary" and vice versa and see the colour change accordingly.

## Screenshots
